noetic/distribution.yaml

```diff
@@ -3658,8 +3658,8 @@ repositories:
       - theora_image_transport
       tags:
         release: release/noetic/{package}/{version}
-      url: https://github.com/ros-gbp/image_transport_plugins-release.git
-      version: 1.14.0-1
+      url: https://github.com/ros2-gbp/image_transport_plugins-release.git
```

Comment:
@ijnek double checking that the image_transport_plugins team maintaining the ROS 1 branches are good with using the ros2-gbp repository for everything. Using ros2-gbp is only currently required for ROS 2 distributions but there is no restriction against using it for ROS 1, from that perspective whatever is easier for the team is what is most important.

Reply:
@nuclearsandwich Thanks for reviewing! All of the ROS 1 maintainers have moved on, so I am somewhat taking over those branches too. I prefer having them all in ros2-gbp to ease maintenance.